core/src/main/scala/chisel3/Data.scala

```diff
@@ -774,31 +693,39 @@ abstract class Data extends HasId with NamedComponent with SourceInfoDoc {
     * Directionality data is still preserved.
     */
   private[chisel3] def cloneTypeFull: this.type = {
-    val clone = this.cloneType.asInstanceOf[this.type] // get a fresh object, without bindings
+    val clone = this.cloneType// get a fresh object, without bindings
     // Only the top-level direction needs to be fixed up, cloneType should do the rest
     clone.specifiedDirection = specifiedDirection
     clone
   }
 
-  /** Connect this $coll to that $coll mono-directionally and element-wise.
+  /** The "strong connect" operator.
+    *
+    * For chisel3._, this operator is mono-directioned; all sub-elements of `this` will be driven by sub-elements of `that`.
+    *  - Equivalent to `this :#= that`
     *
-    * This uses the [[MonoConnect]] algorithm.
+    * For Chisel._, this operator connections bi-directionally via emitting the FIRRTL.<=
+    *  - Equivalent to `this :<>= that`, with the additional restriction that the relative bundle field flips must match
```

Comment:
Doesn't :<>= already have that restriction? Does this operator really have that restriction?